app/views/custom/layouts/_footer.html.erb

```diff
@@ -0,0 +1,38 @@
+<footer>
+  <div class="row">
+    <div class="small-12 large-4 column">
+      <h1 class="logo">
+        <%= link_to "https://pro-ethics.eu", title: t("layouts.footer.framework") do %>
+          <%= image_tag "ProEthics.svg", width: "250px" %>
+        <% end %>
+      </h1>
+
+      <p class="info">
+        <%= sanitize(t("layouts.footer.description",
+          open_source: link_to(t("layouts.footer.open_source"), t("layouts.footer.open_source_url"), target: "blank", rel: "nofollow"),
+          consul:  link_to(t("layouts.footer.consul"), t("layouts.footer.consul_url"), target: "blank", rel: "nofollow"))) %>
+        <%= t("layouts.footer.contact_us") %>
```

Comment:
Since we're customizing the footer, maybe we can remove the "For technical assistance, visit" incomplete text (which CONSUL had by default until recently :sweat_smile:).